feature-group-definitions/font-size-adjust.yml

```diff
@@ -0,0 +1,9 @@
+name: font-size-adjust
+description: The `font-size-adjust` CSS property preserves the apparent text size by adjusting the font size so that the specified metric is scaled to the same size regardless of the font used. This is useful when fallback fonts may be used.
```

Comment:
I like this new description—I feel like I was able to figure out what is going on, which is a first for this property for me. 😅 

I tried a somewhat aggressive revision here, to make it easier to understand the (several!) relationships described.

```suggestion
description: The `font-size-adjust` CSS property preserves apparent text size, regardless of the font used, by scaling fonts to the same size with respect to a specific metric, such as x-height. This can help make fallback fonts look the same size.
```

Reply:
Looks good to me, thanks @ddbeck!